hack/vendor.sh

```diff
@@ -13,7 +13,7 @@ clone git github.com/go-check/check 64131543e7896d5bcc6bd5a76287eb75ea96c673
 clone git github.com/gorilla/context 14f550f51a
 clone git github.com/gorilla/mux e444e69cbd
 clone git github.com/kr/pty 5cf931ef8f
-clone git github.com/mattn/go-sqlite3 b4142c444a8941d0d92b0b7103a24df9cd815e42
+clone git github.com/mattn/go-sqlite3 3b3f1d01b2696af5501697c35629048c227586ab
```

Comment:
Mind switching this to the updated version tag @mattn created for us (`v1.1.0`; https://github.com/mattn/go-sqlite3/pull/235#issuecomment-138137516)? :heart: :heart_eyes: